src/main/java/com/ksm/wstbackoffice/domain/Warehouse.java

```diff
@@ -0,0 +1,27 @@
+package com.ksm.wstbackoffice.domain;
+
+import com.ksm.wstbackoffice.enums.PriceType;
+import com.ksm.wstbackoffice.enums.WarehouseType;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "warehouses")
+public class Warehouse {
```

Comment:
In order to have some many to one relationship let´s add Address class with Country, City, Street, Building number, zip code, and so on.
And add the address as a field to the Warehouse

Reply:
done